Finished Task 2 for Lesson7 RSpec Expectations.

Yulson opened the pull request:
@d-konovalov, please review Main_project_Tests_RSpecMatchers.rb there are tests for Demo.Redmine project with using of RSpec Expectations.

Yulson committed:
Finished Task 2 for Lesson7 RSpec Expectations.

d-konovalov commented on Main_project_Tests_RSpecMatchers.rb:
```diff
@@ -0,0 +1,192 @@
+
+		self_watcher = user.first_name + ' ' + user.last_name
+		actual_watchers = @driver.find_elements(:css, 'ul.watchers').map(&:text)
+		assert_true(actual_watchers.include? self_watcher)
```
should it be ```expect```-like syntax as well?

d-konovalov commented on Main_project_Tests_RSpecMatchers.rb:
```diff
@@ -0,0 +1,192 @@
+		issue3 = create_new_issue(type: '3')
+		     	
+		array_subjects = @driver.find_elements(:class, 'subject').map(&:text)
+		expect(issue3.name).to eql(array_subjects[0])
```
try to create an array with issue names and compare it with ```array_subjects```

Yulson committed:
Merge branch '6_Classes_Inheritance_and_Modules' of https://github.com/Yulson/ruby-automation into 7_RSpec_Expectations

Yulson committed:
Added expect matcher to test_add_self_to_watchers, added expect matcher to compare 2 arrays in test_create_issues

Yulson committed:
Added RSpec tests to Simple_Trello project, according to task 1 of RSpec Expectations Lesson

Yulson committed:
Modified locators in Main_project_Test_RSpecMatchers.rb file

Yulson committed:
Removed odd lines in Simple_RSpec.rb file

Yulson commented:
@d-konovalov, please approve this pull request. I deleted 2 odd lines in tests for Trello project and changed locators in tests for DemoRedmine project.
Thank you in advance.
